client/src/assets/messages/common.ts

```diff
@@ -0,0 +1,7 @@
+// Add any common strings here.
+
+// Titles for the various pages in the app
```

Comment:
I put mine in a class of messages for organization. See prediction messages here: https://github.com/BlueConduit/open-data-platform/pull/80/files#diff-4c8fb56b2ab1f451991024f224e1a2cc85e2519b19dfa59ea18e97ebfeafa450

Reply:
That looks better so I've switched it to be the same.